src/lib/containers/home.tsx

```diff
@@ -85,7 +85,10 @@ export class Home extends React.Component<Props, State> {
   }
 
   componentDidUpdate() {
-    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
+    if (this.currentScrollOffset < 0) {
```

Comment:
So `componentDidUpdate` gets triggered after the refresh network fetch has occurred not immediately after triggering the refresh, right?